site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -178,23 +187,44 @@ class RegisterUser extends Controller
     return redirect('/');
   }
 }
+//..
 ```
 
-_Note: FusionAuth includes several other options for user registration. Be sure to [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/registrations) to make sure you include all the appropriate options._
+Finally, you'll need to create two routes to register new users: one to display the registration form and another to process the data and interact with the FusionAuth client. Add the following lines to your `routes/web.php` file (remove the welcome route):
+
+```php
+Route::view('/register', 'register');
+Route::post('/register', 'RegisterUser');
+```
+
+{% include _callout-note.liquid content=
+"FusionAuth includes several other options for user registration. Be sure to [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/registrations) to make sure you include all the appropriate options."
+%}
 
 At this point, you should be able to access the `/register` route in your Laravel application and successfully register a new user in FusionAuth.
 
-{% include _image.liquid src="/assets/img/blogs/fusionauth-laravel/fusionauth-2020-05-22-a.png" alt="The registration page for a Laravel application using FusionAuth." class="img-fluid" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/fusionauth-laravel/new-user-register.png" alt="The registration page for a Laravel application using FusionAuth." class="img-fluid bottom-cropped" figure=false %}
 
-## Logging a user in
-Now that users can register a new account, you'll need to allow them to login using the email and password they created during registration. This will require two new routes, so open your `routes/web.php` file again and add the following lines:
+{% include _callout-tip.liquid content=
+"Getting errors at this point?  One common mistake is forgetting to import the FusionAuth client into your controller - `use FusionAuth\FusionAuthClient;`"
+%}
 
-```php
-Route::view('/', 'login');
-Route::post('/login', 'LoginUser');
-```
+##  Let's take a look at our progress
+
+Previous to the registration step above, no new users had been added or registered yet.
```

Comment:
Maybe `Before we built out and called the registration functionality above, no new users...`.